02-express-tutorial/logger.js

```diff
@@ -0,0 +1,12 @@
+//logger function is the middleware.It allows for reuse of the function when called
+//Either you send back your own response or you pass it on to the next middleware
+//in the future, better to keep logger ina separate file for easier readibility
+const logger = (req, res, next) => {
```

Comment:
Logger is middleware so it should go in a folder called middleware.  This helps to keep things organized.